### DISCUSSION TITLE
OrientDB with Jakarta EE 9/10

### DISCUSSION THREAD

**Discussion by jacquipre** [orientechnologies/orientdb#9978] [General]
Hi,

since we are updating our dependencies and frameworks in our application, we are just wondering, if there are any plans with OrientDB to be compatible with Jakarta EE 9/10 ?

Thanks and regards,
jacquipre.

**Comment by tglman** [comment]
Hi,

We do not have too much Interaction with Jakarta EE in OrientDB as today, we do depends mainly on the SE, so I do not see any big problem to do some upgrades on that part in future, even though as today there are not much plans on working on that.

Regards

**Reply by jacquipre** [reply]
Thanks for your answer. 

So how can we get this topic to the OrientDB roadmap? Can I open an issue?

Regards,
jacquipre
